docs/migration/enact/migrating-to-enact-5.md

```diff
@@ -0,0 +1,217 @@
+}
+```
+
+#### Removed deprecated React APIs and React DOM APIs
+* **`propTypes` and `defaultProps` for functions**: These are removed. Using `propType` will be silently ignored. Use ES6 default parameters instead of `defaultProps`. Class components will continue to support `defaultProps`.
+* **`ReactDOM.findDOMNode`**: This API is removed. Replace it with DOM refs.
+* **`react-dom/test-utils`**: Import `act` from `react` instead of `react-dom/test-utils`.
+
+For more details, please refer to [React 19 Upgrade Guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#breaking-changes) and [React v19](https://react.dev/blog/2024/12/05/react-19) from React Blog.
+
+### cli
+`@enact/cli` must be upgraded to version `7.0.0-alpha.6` or newer, as shown below.
+
+```sh
+npm install -g @enact/cli@next
+```
+
+
+
+### webOS TV
+
+Enact 5.0 no longer supports the 2025 TV platform or earlier versions.
+
+## limestone
+
+### General
+
+Limestone now uses the design-tokens system to style its components, importing alias tokens from the [design-tokens](https://github.com/enactjs/design-tokens). This enables consistent and customizable styling across all components. *Note that the names of the design tokens have not been determined yet so they can be changed in the future.*
+
+With the removal of the `ReactDOM.findDOMNode` API, `ContextualPopupDecorator` and `Dropdown` have been changed to include a sibling DOM node by using `@enact/core/internal/WithRef` to access DOM nodes.
+
+#### Tips
+
+Due to introducing `@enact/core/internal/WithRef` HoC, one of your apps can face unexpected issues. We have found 2 sample cases that you should know in advance.
+
+##### `ContextualPopupDecorator`
+Make sure your component wrapped with `ContextualPopupDecorator` passes the `rest` props to the inner component to get the proper ref, as shown below.
+
+```js
+const SampleButton = kind({
+  render: (props) => {
+    const {icon, ...rest} = props;
+
+    return (
+      <Icon
+        {...rest} // Ensure the inner component receives the `rest` props
+        //...
+      >{icon}</Icon>
+    );
+  }
+});
+
+const ContextualSampleButton = ContextualPopupDecorator(SampleButton);
+```
+
+##### `Panels`
+Make sure the root element of your `Panel` component inside `Panels` is not a `<>` (fragment). It will result in incorrect panel transition animations.
+
+
+### `Alert`
+The prop `overlayPosition` is added to set the position of the alert when the type is `overlay`.
+
+### `Button`
+The prop `bordered` and `centered` are added.
+
+### `Card`
+The new component `Card` is added.
+
+### `Chip`
+The new component `Chip` is added.
+
+### `KeyGuide`
+The image-based `keyGuide` is added.
+
+### `Switch`
+The Switch icon is now drawn using CSS instead of an icon font.
+
+### `TabLayout`
+Tabs in horizontal `TabLayout` now support scrolling.
+
+### Style
+Various styling changes have been made to support new UI/UX designs for the webOS TV platform.
+
+#### Colors
```

Comment:
Hmm... below looks awkward since @lime-XXX is not for @sand-XXXX... 
So, If I am trying to migrate from sandstone, these may be confused.

Reply:
I assume that they were changed 'sand' to 'lime' when they are reading this. I don't mind if we change it to 'sand'.